bootloader/README.md

```diff
@@ -1,123 +1,41 @@
+  ```none
+  reset halt
+  flash protect 0 0 off
+  ```
 
- * In the windows utility set device -> settings to "connect under reset" true! (if you can't find this that's okay)
- * Do a full erase of your core, do this a few times, if you're in the windows utility, view the memory and make sure it's zero-ed out.
- * Write your bootloader.bin back to 0x08000000
- * Congrats, you re-wrote your bootloader!
-
-**3. Lock your bootloader back up! (Optional, but recommended)**
-
- * Grab the locking firmware here:  (https://github.com/spark/firmware/tree/master/bootloader/tools (use "raw")
- * dfu that to `dfu-util -d 1d50:607f -a 0 -s 0x08005000:leave -D locker-firmware.bin`
- * your core should reset, flash some lights, and flash 'red' when locked.
-
-**4. Okay, lets keep going and update the radio driver on the CC3000**
-
- * Grab the patch programmer from here: https://github.com/spark/cc3000-patch-programmer/blob/master/build/cc3000-patch-programmer.bin?raw=true
- * dfu it with: `dfu-util -d 1d50:607f -a 0 -s 0x08005000:leave -D cc3000-patch-programmer.bin`
- * hold down the mode button for 3+ seconds until the light starts blinking, let your core sit for 20-60 seconds after the light stops blinking.
- * At this point you can erase your core with the st-link programmer (did you lock your bootloader?), to get it back into dfu mode
-
-
-### It's time to write some stuff to external flash!
-
- **1. Lets make some keys (optional step if your keys have been fine):**
-
-```
-    openssl genrsa -out core.pem 1024
-    openssl rsa -in core.pem -pubout -out core_public.pem
-    openssl rsa -in core.pem -outform DER -out core_private.der
-```
-
- **2. Flash your new private key:**
-
-    `dfu-util -d 1d50:607f -a 1 -s 0x00002000 -v -D core_private.der`
-
- **3. Grab the server public key, and flash it:**
-    https://s3.amazonaws.com/spark-website/cloud_public.der
-
-    `dfu-util -d 1d50:607f -a 1 -s 0x00001000 -v -D cloud_public.der`
-
- **4. Grab the latest firmware and flash it**
-
-    https://github.com/spark/firmware/blob/master/main/build/core-firmware.bin?raw=true
+- in the bootloader directory run
 
-```
-    # to the factory reset spot
-    dfu-util -a 1 -s 0x00020000 -v -D core-firmware.bin
+  ```none
+  make PLATFORM=photon all program-openocd
+  ```
 
-    ## to the main firmware spot
-    dfu-util -a 0 -s 0x08005000:leave -D core-firmware.bin
-```
+- finally send the telnet command
 
- **5. If you made new keys, be sure to send someone at Spark your new public key, along with your core id, otherwise your core won't connect to the cloud again!**
+  ```none
+  flash protect 0 0 on
+  ```
 
- *(Thanks to David for putting this together)*
- Ref: https://community.spark.io/t/st-link-programmer-shield-only-so-youve-decided-you-want-to-update-your-bootloader-and-everything/2355
+> *(Thanks to David for putting this together)*
```

Comment:
Can probably be removed as well since it talks about just Core.

Reply:
The whole `README.md`?

Reply:
Good catch! I thought I had a filter to protect against these directories.